src/Resources/contao/config/config.php

```diff
@@ -10,8 +10,9 @@
  * @license LGPL-3.0-or-later
  */
 
-$GLOBALS['TL_HOOKS']['loadFormField'][] = ['inspiredminds.fieldsetduplication.listener.formhook', 'onLoadFormField'];
+$GLOBALS['TL_HOOKS']['loadFormField'][]     = ['inspiredminds.fieldsetduplication.listener.formhook', 'onLoadFormField'];
 $GLOBALS['TL_HOOKS']['compileFormFields'][] = ['inspiredminds.fieldsetduplication.listener.formhook', 'onCompileFormFields'];
+$GLOBALS['TL_HOOKS']['prepareFormData'][]   = ['inspiredminds.fieldsetduplication.listener.formhook', 'onPrepareFormData'];
```

Comment:
Did you run the CS fixer? I think these whitespaces will get removed then.

Reply:
Now, I've done it.